### DISCUSSION TITLE
v3.2.0-beta.1 - Multi MPPT changes for monophase inverters

### DISCUSSION THREAD

**Discussion by alexdelprete** [alexdelprete/ha-abb-powerone-pvi-sunspec#88] [Announcements]
This release contains changes for monophase + multi MPPTs and restructures the sensors definition to make it more flexible.
Solves #69 (wrong DC value of -0.0 for monophase + multiple MPPTs setups)

If you have a monophase inverter with multiple strings, please let us have a feedback in Discussions section, every release can be discussed in the Announcements category. You can also give the feedback in HA forums [thread](https://community.home-assistant.io/t/custom-component-abb-power-one-fimer-pv-inverters-sunspec-modbus-tcp).

Thanks to @ivanfmartinez for his contribution.

<hr /><em>This discussion was created from the release <a href='https://github.com/alexdelprete/ha-abb-powerone-pvi-sunspec/releases/tag/v3.2.0-beta.1'>v3.2.0-beta.1 - Multi MPPT changes for monophase inverters</a>.</em>

**Comment by alexdelprete** [comment]
@ivanfmartinez I'm testing it since a month, it's working fine here with 2 MPPTs on my 3-phase inverter. Is it working there? I'd like to release v3.2 final.

Let me know, thanks.

**Comment by ivanfmartinez** [comment]
Its working fine on my 2 inverters , both single-phase with 2 mppts.

**Reply by alexdelprete** [reply]
Good. I released v3.2.0.

Thanks.
